package.json

```diff
@@ -5,7 +5,9 @@
   "license": "MIT",
   "scripts": {
     "postinstall": "nx run db:generate",
-    "prepare": "husky install"
+    "prepare": "husky install",
+    "serve:backend": "nx serve backend",
```

Comment:
The devcontainer npx feature was not reliable here for some reason so I opted to have the container use these npm scripts as a wrapper.